scripts/lib/api/updateLinks.test.ts

```diff
@@ -84,6 +115,25 @@ describe("updateLinks", () => {
         },
       ]
     `);
+    expect(objectsInv.entries.map((e) => e.uri)).toMatchInlineSnapshot(`
```

Comment:
```suggestion
    expect(objectsInv.entries.map((e) => e.uri)).toEqual(`
```

Reply:
Inline snapshot is correct here as it's a string. Would you like to me to change to a list? I used the snapshots as I was under the impression they were easier to copy-paste when they need updating.

Reply:
Ok sounds good, fixed in ac85d66ce80a7efdb2aa13bd84a0f2b93ef3cb86